ui/packages/consul-ui/translations/common/en-us.yaml

```diff
@@ -38,6 +38,7 @@ consul:
   destinationname: Destination Name
   sourcename: Source Name
   displayname: Display Name
+  secretID: Secret ID
```

Comment:
Could [components/consul/en-us.yaml](https://github.com/hashicorp/consul/blob/e93e5ce149b50a336f22e592e7cb5d3fc8cca12d/ui/packages/consul-ui/translations/components/consul/en-us.yaml#L47) be a possible alternative for this to tag usage for a token copyable text? Something like: `{{t "components.consul.token.secretID"}}` or `{{t "components.consul.token.properties.secretID"}}`

Reply:
🤔 yes, it would be better under tokens. I thought SecretID was used in other ACL models, but looks like it's just in tokens. I'll move that over.